util/src/main/scala/org/ensime/util/path.scala

```diff
@@ -57,6 +57,13 @@ package object path {
       require(path.isDirectory && to.isDirectory)
       Files.walkFileTree(path, EnumSet.of(FileVisitOption.FOLLOW_LINKS), Integer.MAX_VALUE, new CopyDirVisitor(path, to))
     }
+    def copyFileTo(to: Path): Unit = {
+      require(path.isFile)
+      if (!Files.exists(to)) {
+        Files.createDirectories(to.getParent())
```

Comment:
looks like it works well if the parent directory already exists, but maybe is a good idea to add test for this case?